microsoft/testsuites/dpdk/dpdkvpp.py

```diff
@@ -104,7 +105,18 @@ def _install(self) -> bool:
                 "Could not install vpp with fdio provided installer"
             ),
         )
-        node.os.update_packages("")
+        node.os.get_repositories()
```

Comment:
What this line actually do? If it's to trigger some inner actions, the actions should be exposed to get logic clear.

Reply:
The old way of calling update_packages with no argument was a hack, this other call is the interfaces proper way to make that call.

Reply:
I think it has already been covered in is_package_in_repo?

```
    def is_package_in_repo(self, package: Union[str, Tool, Type[Tool]]) -> bool:
        """
        Query if a package/tool exists in the repo
        Return Value - bool
        """
        package_name = self.__resolve_package_name(package)
        if self._first_time_installation:
            self._initialize_package_installation()
            self._first_time_installation = False
        return self._is_package_in_repo(package_name)
```

Reply:
I don't think that is guaranteed to work if another package has been installed before adding the new repository info. Since we're adding a new repository with the install script, the instructions suggest running apt update/zypper renew etc. to make sure the repo info is added correctly.

In the event the test changes, a tool is installed which installs a package, or a dependency changes, I think it's better to explicitly ask for the repository cache update rather than rely on the side effect of the package_in_repo function.